docs/source/contents/4_draft_mapping.rst

```diff
@@ -1784,6 +1784,271 @@ subject
+
+Justification
+^^^^^^^^^^^^^
+For values *outside* of the following table, we selected the `edm:hasType` property as it aligns well with the possible overlap between `mods:genre` and `mods:physicalDescription/form`. To help prevent duplicating string literals and URIs, the following table suggests a mapping for a limited subset of the union of values in `genre[not(@*)]` and `genre[@authority='dct']`.
+
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| (//genre[not(@*] | //genre[@authority='dct']) | RDF Predicate | URI                                              | dcterms text value |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| cartographic                                  | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/car> | Cartographic       |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| image                                         | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/img> | Image              |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| notated music                                 | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/not> | Notated music      |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| still image                                   | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/img> | Still image        |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| text                                          | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/txt> | Text               |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+| *                                             | edm:hasType   | <http://www.europeana.eu/schemas/edm/>           | N/A                |
++-----------------------------------------------+---------------+--------------------------------------------------+--------------------+
+
+XPaths
+^^^^^^
+:code:`genre[not(@*)][string() = 'cartographic']` OR
+:code:`genre[not(@*)][string() = 'notated music']` OR
+:code:`genre[@authority = 'dct'][string() = 'image']` OR
+:code:`genre[@authority = 'dct'][string() = 'still image']` OR
+:code:`genre[@authority = 'dct'][string() = 'text']` OR
+:code:`genre[not(@*) and not(string() = ('cartographic','notated music'))]`
```

Comment:
I guess I didn't notice this one on the end. These are the edm:hasType values. Is there any reason why we can't just make this a separate category instead of putting it in with dcterms:type? I assume that everything in the same category is treated the same way as everything else in the category. Maybe this is a faulty assumption on my part, but I think it makes interpreting this doc easier.